internal/pkg/scorecard/helpers/versions.go

```diff
@@ -22,11 +22,11 @@ import (
 
 const v1alpha1 = "v1alpha1"
 const v1alpha2 = "v1alpha2"
-const DefaultScorecardVersion = v1alpha1
+const DefaultScorecardVersion = v1alpha2
 const LatestScorecardVersion = v1alpha2
 const VersionOpt = "version"
 
-var ScorecardVersions = []string{DefaultScorecardVersion, LatestScorecardVersion}
+var ScorecardVersions = []string{v1alpha1, DefaultScorecardVersion, LatestScorecardVersion}
```

Comment:
Now that `DefaultScorecardVersion` and `LatestScorecardVersion` are the same, do we need both?

Would it make sense to use this:
```suggestion
var ScorecardVersions = []string{v1alpha1, v1alpha2}
```

Reply:
yep, I'll make that change.